### PR TITLE
#1001 added link to production, updated production template

### LINKED ISSUE #1001: Can't add supplementary files to articles loaded through back content plugin
**Describe the bug**
When an article is loaded via the back content plugin, it is not possible to add supplementary files to it. Instead, one must manually move the article back into production through the admin.

**Janeway version**
1.3.3.1

### PR DESCRIPTION
**Pull request by ajrbyers**
updated view to assign user as PM.

Closes #1001

### CODE REVIEW

File: src/production/views.py [hunk 1]
```diff
@@ -158,7 +158,29 @@ def production_article(request, article_id):
...
-    production_assignment = models.ProductionAssignment.objects.get(article=article)
+
+    try:
+        production_assignment = models.ProductionAssignment.objects.get(
+            article=article
+        )
+    except models.ProductionAssignment.DoesNotExist:
+        # It is assumed that if a user enters this URL by hand and passes
+        # the decorator that they are fine being assigned as the PM
+        production_assignment = models.ProductionAssignment.objects.create(
```
Comment (mauromsl) [inline src/production/views.py]: Why is this needed? Seems a weird thing to create this objects on a GET request

Comment (ajrbyers) [reply src/production/views.py]: I agree it is a little odd, but if an article doesn't directly go through the Production process (ie. through the back content plugin) folks still want access to some of the controls from here. Its a one-to-one as well so only a single entity can exist so I think its okay. Open to alternative suggestions though.

Comment (mauromsl) [reply src/production/views.py]: If that is the problem, what about creating the production assignment at the time of loading the article from the back content plugin? It seems seem to be what we are implicitly doing here.

Comment (ajrbyers) [reply src/production/views.py]: Yeah that is a _specific_ use case, but i meant _any_ article that doesn't go through the production workflow element, so I decided it was best to simple handle it here.

Comment (mauromsl) [reply src/production/views.py]: Yeah but this doesn't explicitly target "any article that doesn't go through the production workflow element", but rather any article for which we haven't found a production assignment.

Comment (ajrbyers) [reply src/production/views.py]: These are one and the same. An article can't go through production without a Production Assignment.

Comment (mauromsl) [reply src/production/views.py]: Then we should at least render a little form for the user to confirm this.
